aboutcode-data/README.rst

```diff
@@ -621,19 +622,19 @@ For example:
 -  A tool may prefer to provide data with products or components as top level
    objects. The components used in a Product are naturally embedded in the products.
 
--  A tool may be concerned more with files will provide files as top
-   level objects and may embed packages details when they are found for
+-  A tool concerned more with files, will provide files as top
+   level objects and may embed package details when they are found for
    a file or directory path.
 
 -  Another tool may focus on packages and provide packages first with
-   components references and possibly embedded files. A matching tool
+   component references and possibly embedded files. A matching tool
    may provide packages first and reference matched files. The file
    paths of a package are naturally embedded in the package, though
    using references may help keep the data simpler when there is a large
-   volume of files
+   volume of files.
 
--  A tool that generates attribution documentation may be interested
-   first by components and second by licenses or packages references.
+-  A tool that generates attribution documentation may interest
```

Comment:
Well it is no mine either. Let's go with:
`A tool that generates attribution documentation may focus first on components and second on licenses or packages references.`